src/remotepublish.cmd.js

```diff
@@ -155,6 +158,30 @@ class RemotePublishCommand extends AbstractCommand {
     return this;
   }
 
+  withCustomVCL(value) {
+    if (value && value.length > 0) {
```

Comment:
from an API perspective, I think this should also allow adding single VLCs and update the internal `this._vcl`.
otherwise, rename it to `withCustomVCLs()`

Reply:
Since the parameter `custom-vcl` (list of file paths) has to be transformed into an object (name:contentoffile), I initially thought we could have the 2 methods to have the 2 ways of providing the data: either the list of files or the object directly. But you are right, this is confusing and introduce useless features. I removed the second method.